sentry_sdk/integrations/arq.py

```diff
@@ -72,12 +71,10 @@ def patch_enqueue_job():
 
     async def _sentry_enqueue_job(self, function, *args, **kwargs):
         # type: (ArqRedis, str, *Any, **Any) -> Optional[Job]
-        hub = Hub.current
-
-        if hub.get_integration(ArqIntegration) is None:
+        if sentry_sdk.get_client().get_integration(ArqIntegration) is None:
             return await old_enqueue_job(self, function, *args, **kwargs)
```

Comment:
We can use `ensure_integration_enabled_async` here.

Reply:
Seems to work in this integration -- changed